lib/engine/game/g_1893/minor.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require_relative '../../minor'
+
+module Engine
+  module Game
+    module G1893
+      class Minor < Engine::Minor
```

Comment:
minors don't have shares can we delete this class

Reply:
Has been deleted.